libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -486,6 +486,11 @@ void AP_Airspeed::calibrate(bool in_startup)
         if (in_startup && param[i].skip_cal) {
             continue;
         }
+        if (sensor[i] == nullptr) {
+            GCS_SEND_TEXT(MAV_SEVERITY_INFO,"Airspeed %u not initalized, cannot cal", i+1);
+            state[i].cal.start_ms = 0;
```

Comment:
Do need this? If we do it should be up between 487 and 479 so that we reset it for all the cases that `continue`

Reply:
if they continue, they dont do a cal...have to init the start_ms, otherwise, no cal occurs

Reply:
so the GCS command does not initialize the time, so should be able to remove the line...will test tomorrow, hopefully